SkietbaanBE/SkietbaanBE/Migrations/20190206075919_MagicMigration.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace SkietbaanBE.Migrations
+{
+    public partial class MagicMigration : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+        
```

Comment:
Please remove this whitespace

Reply:
@ash-rabbit thats an autogenerated file 😆